.eslintrc.js

```diff
@@ -50,7 +50,8 @@ const config = deepMerge(defaultConfig, {
         '@typescript-eslint/no-unsafe-return': 0,
         '@typescript-eslint/no-unsafe-call': 0,
         '@typescript-eslint/no-empty-interface': 0,
-        '@typescript-eslint/restrict-plus-operands': 0
+        '@typescript-eslint/restrict-plus-operands': 0,
+        '@typescript-eslint/no-unnecessary-type-assertion': 0
```

Comment:
I do not add a sweeping rule hiding issues across the code base to fix one error. Please use `// eslint-disable-next-line @typescript-eslint/no-unnecessary-type-assertion`

Reply:
Ah thanks! I didn't know that is possible.